### ISSUE TITLE
SQLite unparsable FTS5 `content` option

### ISSUE THREAD

**Issue by rvnkcode** [sqlfluff/sqlfluff#6944]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

Maybe related to this issue, https://github.com/sqlfluff/sqlfluff/issues/6403
When set the `content` option for FTS5, I've got `Found unparsable section` error.
Ref: https://sqlite.org/fts5.html#external_content_and_contentless_tables

### Expected Behaviour

Parsing and linting passes successfully

### Observed Behaviour

```
== [stdin] FAIL
L:   1 | P:  35 |  PRS | Line 1, Position 35: Found unparsable section: "(a, b,
                       | c, content='');"
WARNING: Parsing errors found and dialect is set to 'sqlite'. Have you configured your dialect correctly?
All Finished 📜 🎉!
```

### How to reproduce

```sh
echo "CREATE VIRTUAL TABLE ft USING fts5(a, b, c, content='');" | sqlfluff lint --dialect=sqlite -
```

### Dialect

SQLite

### Version

3.4.0

### Configuration

```
[sqlfluff]
max_line_length = 120
```

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
